packages/front-end/pages/attributes.tsx

```diff
@@ -53,25 +66,18 @@ const FeatureAttributesPage = (): React.ReactElement => {
                     />
                   </Tooltip>
                 </th>
+                <th>Archived</th>
               </tr>
             </thead>
             <tbody>
               {attributeSchema && attributeSchema.length > 0 ? (
                 <>
-                  {attributeSchema.map((v, i) => (
-                    <tr key={i}>
-                      <td className="text-gray font-weight-bold">
-                        {v.property}
-                      </td>
-                      <td className="text-gray">
-                        {v.datatype}
-                        {v.datatype === "enum" && <>: ({v.enum})</>}
-                      </td>
-                      <td className="text-gray">
-                        {v.hashAttribute && <>yes</>}
-                      </td>
-                    </tr>
-                  ))}
+                  {attributeSchema
+                    .filter((o) => o?.archived !== true)
+                    .map((v, i) => drawRow(v, i))}
+                  {attributeSchema
+                    .filter((o) => o?.archived === true)
+                    .map((v, i) => drawRow(v, i))}
```

Comment:
To simplify this code, I would create a new attribute array that is ordered the way you want.  Then, we could keep this code as a single `map` call.

```ts
const orderedAttributes = [
  ...attributeSchema.filter(o => !o.archived),
  ...attributeSchema.filter(o => o.archived)
]
```